configs/benchmarks/mmdetection/_base_/default_runtime.py

```diff
@@ -7,7 +7,7 @@
         # dict(type='TensorboardLoggerHook')
     ])
 # yapf:enable
-custom_hooks = [dict(type='NumClassCheckHook')]
+# custom_hooks = [dict(type='NumClassCheckHook')]
```

Comment:
This file hasn't been updated to the 2.0 version

Reply:
done